android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BluetoothSite.java

```diff
@@ -228,7 +228,16 @@ private File getHtmlFile() {
     return new File(websiteDir, "website.html");
   }
 
+  private File getTempFile() {
+    File websiteDir = new File(activity.getFilesDir(), "Websites");
+    websiteDir.mkdir();
+    return new File(websiteDir, "temp.html");
```

Comment:
Hm, you might need to set a cache-path? https://developer.android.com/reference/android/support/v4/content/FileProvider.html